dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/cookies/SameSiteCookieUtils.java

```diff
@@ -0,0 +1,271 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
```

Comment:
Maybe a question for @pnicolucci . 


This code in this file looks to based off the pseudocode example provided in https://www.chromium.org/updates/same-site/incompatible-clients/   (which is licensed as Apache-2.0) 

Should we update this to be an Apache 2.0 License or keep it as EPL 2.0?

Reply:
Vlad and I talked about this and a comment acknowledging that we pulled from the pseudocode should suffice.